src/redirects.js

```diff
@@ -160,6 +165,14 @@ const datasetRedirectPatterns = [
   ['/monkeypox/hmpxv1', '/mpox/clade-IIb'],
   ['/monkeypox/hmpxv1/big', '/mpox/lineage-B.1'],
   ["/monkeypox", "/mpox"],
+
+   /**
+     * We shifted avian-flu and seasonal-flu to top level to stop nesting them
+     * both under flu/ so that avian-flu is more discoverable
```

Comment:
Suggestion: Update doc references 0befac71c26dacbdfc9a42412b11f77c93100049

Reply:
Thanks! I cherry-picked https://github.com/nextstrain/nextstrain.org/commit/0befac71c26dacbdfc9a42412b11f77c93100049 into this PR